tests/unit/serializers/drf-test.js

```diff
@@ -1,14 +1,82 @@
 import { module, test } from 'qunit';
 import { setupTest } from 'ember-qunit';
+// import sinon from 'sinon';
```

Comment:
Can we take this off if it's not required?

Reply:
no it is for reference for skipped test